cinema/views.py

```diff
@@ -48,6 +74,32 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    def get_queryset(self):
+        movie = self.request.query_params.get("movie")
+        date = self.request.query_params.get("date")
+        queryset = self.queryset
+
+        if date:
+            queryset = queryset.filter(show_time__date=date)
+
+        if movie:
+            queryset = queryset.filter(movie=movie)
```

Comment:
im not sure this filter will work with multiple movie ids, wouldnt it be better to use filter like with genres and actors?

Reply:
We are filtering by ?actors=, ?genres= and ?title= in first case. In second case by ?date=...&movie=1, and in readme task is "Implement filtering for movie sessions by date and movie" so for me it make sence to leave it like this. But task its tricky and not exactly clear, lets see what mentor says.